### ISSUE TITLE
Hacking the Input of Quotes

### ISSUE THREAD

**Issue by ahaghgoo** [crsmithdev/hint#2]
I know that adding custom quotes is on your todo list, but I'm wondering if there's currently a way to do  it without that feature being implemented?  I tried just inputting the quote into the excel file where the quotes are listed but I'm assuming I overlooked a few steps to get them to show. 

Thanks for the consideration!  Love the app!!

**Comment by crsmithdev** [comment]
Yeah, i need to just get around to doing this.  At the moment, you should be able to just replace quotes in a file (or add more, etc.) and build, you'll then have a binary that has the quotes you want in it.
